### ISSUE TITLE
Error when adding ` import {ChartDot, ChartPath, ChartPathProvider} from '@rainbow-me/animated-charts'; `  TypeError: _ReanimatedModule.default.createNode is not a function (it is undefined), js engine: hermes 

### ISSUE THREAD

**Issue by JegoLazaro** [rainbow-me/react-native-animated-charts#98]
TypeError: _ReanimatedModule.default.createNode is not a function (it is undefined), js engine: hermes
Invariant Violation: "main" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called., js engine: hermes

**Comment by lmartinez63** [comment]
Same issue here  this is my package json
{
  "name": "test",
  "version": "1.0.0",
  "main": "node_modules/expo/AppEntry.js",
  "scripts": {
    "start": "expo start",
    "android": "expo start --android",
    "ios": "expo start --ios",
    "web": "expo start --web"
  },
  "dependencies": {
    "@rainbow-me/animated-charts": "^1.0.0-alpha.6",
    "expo": "^49.0.0",
    "expo-status-bar": "~1.6.0",
    "react": "18.2.0",
    "react-native": "0.72.5",
    "react-native-reanimated": "~3.3.0"
  },
  "devDependencies": {
    "@babel/core": "^7.20.0"
  },
  "private": true
}

**Comment by Luyolop1** [comment]
Same here "_ReanimatedModule.default.createNode is not a function (it is undefined), js engine: hermes
"

**Comment by JegoLazaro** [comment]
Hello, for this one the work around that worked for me is to use `react-native-wagmi-charts` instead. Cant seem to find a fix with using animated charts.

**Comment by Luyolop1** [comment]
Yes Im trying to use other chart and now React native chart kit i don 't
know i console.logged my api fetched data and its shows but it cannot be
displayed on chart
it says its having problems converting it im working on that  now
thank you for the feedback
Are the any tutorial that show how to map Line Chart data

**Comment by marioo-lab** [comment]
This error is a showstopper and is a pity, as there are no other good alternatives on react native. Can anyone help fix this and bring this project back to life?

**Comment by Luyolop1** [comment]
Please 🙏🥺 projects can't be finished coz of this!

**Comment by JegoLazaro** [comment]
> This error is a showstopper and is a pity, as there are no other good alternatives on react native. Can anyone help fix this and bring this project back to life? 

Hello, i did a simple crypto price tracker but i encountered this issue. I removed the imports from this package and uninstalled the whole package. I used react native wagmi charts instead and i finished my project. Wagmi charts has good docs also and works the same in my case. As long as this error is not resolved i would wagmi charts is the way to go as of now. Hope this helps 👍🏻

**Comment by marioo-lab** [comment]
> > This error is a showstopper and is a pity, as there are no other good alternatives on react native. Can anyone help fix this and bring this project back to life?
> 
> Hello, i did a simple crypto price tracker but i encountered this issue. I removed the imports from this package and uninstalled the whole package. I used react native wagmi charts instead and i finished my project. Wagmi charts has good docs also and works the same in my case. As long as this error is not resolved i would wagmi charts is the way to go as of now. Hope this helps 👍🏻

Hi! Thanks for the suggestion, but I already tried wagmi charts and it's not as good and also has some animation problems with transitions. It seems something changed in the React Reanimated libarary and old projects didn't update

**Comment by Luyolop1** [comment]
Yeah for real and even now I still haven't found a proper chart that can
display out easily like this one, since I saw it on in a video.Trading view
is not easy as well as I'm trying to make a mobile app.Please anyone help
us out here

**Comment by aniketfuryrocks** [comment]
+1
